livekit-agents/livekit/agents/llm/chat_context.py

```diff
@@ -50,15 +50,15 @@ def create_tool_from_called_function(
         if not called_function.task.done():
             raise ValueError("cannot create a tool result from a running ai function")
 
-        content = called_function.result
-        if called_function.exception is not None:
-            content = f"Error: {called_function.exception}"
+        content = called_function.task.result()
```

Comment:
@theomonnom FYI this path doesn't have test coverage

Reply:
Yep will add a test ty